docs/scarpet/api/Entities.md

```diff
@@ -801,6 +801,7 @@ defining the callback with `entity_event`.
 The following events can be handled by entities:
 
 *   `'on_tick'`: executes every tick right before the entity is ticked in the game. Required arguments: `entity`
+*   `'on_move'`: executes every time an entity changes position, at the end of the `setPos()` method. Required arguments: `entity, velocity, pos1, pos2`
```

Comment:
I suggest not include implementation details in the documentation, but add information that might be important to the implementer
```suggestion
*   `'on_move'`: executes every time an entity changes position, invoked just after it has been moved to the new position. Required arguments: `entity, velocity, pos1, pos2`
```